src/definitions/modules/progress.js

```diff
@@ -23,6 +23,20 @@ window = (typeof window != 'undefined' && window.Math == Math)
     : Function('return this')()
 ;
 
+function toArray(element) {
+  return $.isArray(element)
```

Comment:
Please use native `Array.isArray` instead, we recently removed the jquery variant in the entire codebase, because it's deprecated and not necessary anymore

Reply:
Changed to `Array.isArray` in https://github.com/fomantic/Fomantic-UI/pull/582/commits/be86a0d08cf9e4050e3135911dc39b02fe4565a3